NBXplorer/Repository.cs

```diff
@@ -321,13 +315,14 @@ public NBXplorerNetwork Network
 		}
 
 		DBriizeTransactionContext _TxContext;
-		internal Repository(DBriizeEngine engine, NBXplorerNetwork network, KeyPathTemplates keyPathTemplates)
+		internal Repository(DBriizeEngine engine, NBXplorerNetwork network, KeyPathTemplates keyPathTemplates, RPCClient rpcClient)
```

Comment:
I was considering it, but this is not needed anymore. I am importing in the GetUnused inside the controller. This create less changes.

Reply:
This is needed to unblind a transaction not just to import though

Reply:
no, you can just unblind in the Behavior and only save the unblinded transaction  in the DB

Reply:
wait nvm, cleaning it